dotnet/src/Microsoft.AutoGen/Agents/Agents/IOAgent/ConsoleAgent/IHandleConsole.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // IHandleConsole.cs
 
+using Google.Protobuf;
```

Comment:
don't like that this leaks into the handler - is there another way?

Reply:
maybe it doesn't matter, it will always be in the AgentBase

Reply:
ok